addons/account/static/src/components/section_state/section_state.js

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@odoo/owl';
+import { registry } from '@web/core/registry';
+import { standardWidgetProps } from '@web/views/widgets/standard_widget_props';
+
+const COLLAPSE_ICONS = {
+    collapse_composition: "fa fa-compress",
+    collapse_prices: "fa fa-eye-slash"
+}
```

Comment:
```suggestion
export const COLLAPSE_ICONS = {
    collapse_composition: 'fa fa-compress',
    collapse_prices: 'fa fa-eye-slash',
}
```
Linting comma and quotes + we can also add export it we want to use it somewhere 🤔